exercises/parallel-letter-frequency/README.md

```diff
@@ -33,11 +33,11 @@ If you are using rustup.rs:
 rustup run nightly cargo bench
 ```
 
-- [Benchmark tests](https://doc.rust-lang.org/book/benchmark-tests.html)
+- [Benchmark tests](https://doc.rust-lang.org/1.1.0/book/benchmark-tests.html)
```

Comment:
Is there a particular reason for using the "1.1.0" edition of the book, instead of the generic first-edition link? (<https://doc.rust-lang.org/book/first-edition/>)?

Reply:
No particular reason - would you agree with [https://doc.rust-lang.org/book/first-edition/testing.html](https://doc.rust-lang.org/book/first-edition/testing.html) as the right place to link to?

Reply:
That link doesn't appear to contain the word "benchmark" at all, making it IMO a poor choice for a link purporting to describe benchmark tests. 

Oddly enough, I can't currently find any current first- or second- edition chapter which talks about benchmark tests. The most current version I could find is <https://doc.rust-lang.org/1.16.0/book/benchmark-tests.html>; apparently the chapter was removed without replacement in 1.17.0. 

My advice for now would be to provisionally use the 1.16.0 link for now without giving up the search for something current.

Reply:
Aha, found the correct link for this: <https://doc.rust-lang.org/stable/unstable-book/library-features/test.html>. Please use this one.